packages/compass-e2e-tests/helpers/test-runner-global-fixtures.ts

```diff
@@ -76,7 +71,7 @@ export async function mochaGlobalSetup(this: Mocha.Runner) {
 
     debug('X DISPLAY', process.env.DISPLAY);
 
-    if (!DISABLE_START_STOP) {
+    if (!context.disableStartStop) {
```

Comment:
now that the params are camelCase, I'd kind of expect them to also follow the naming convention and this bool to be `isStartStopDisabled`. might be just me and others would expect the naming to be direct derivative of the original param 🤷 What do you think? If we wanted to rename them, I think we could do that with yargs middleware callback https://yargs.js.org/docs/#api-reference-middlewarecallbacks-applybeforevalidation

Reply:
I think being able to map arguments that are passed to the script to the usage in the code is more important for the dev code like the e2e tests. Middleware would add one more extra step to this process and it doesn't extend the types, so it would be even extra code in between to align the types with the remapping. That's why I'd prefer not to change it